content/v2.0/reference/storage-engine.md

```diff
@@ -0,0 +1,174 @@
+---
+title: InfluxDB storage engine
+description: >
+  An overview of the InfluxDB storage engine architecture.
+weight: 7
+menu:
+  v2_0_ref:
+    name: Storage engine
+v2.0/tags: [storage engine, internals, platform]
+---
+
+The InfluxDB storage engine ensures the following three things:
+
+- Data is safely written to disk
+- Queried data is returned complete and correct
+- Data is accurate (first) and performant (second)
+
+This document details the internal workings of the storage engine.
+This information is presented both as a reference and to aid those looking to maximize performance.
+
+Major topics include:
+
+* [Write Ahead Log (WAL)](#write-ahead-log-wal)
+* [Time-Structed Merge Tree (TSM)](#time-structured-merge-tree-tsm)
+* [Time Series Index (TSI)](#time-series-index-tsi)
+
+{{% note %}}
+##### At a glance: changes to the InfluxDB storage engine from 1.x to 2.0
+- The InfluxDB 2.0 storage engine no longer partitions data into shards by time.
+- **Buckets** replace databases and retention policies.
+- Only TSI is used. There is no more in-memory index.
+- The configuration interface and options have changed. Configuration options are not currently exposed, but will be.
+
+Read about the [v1 storage engine](https://docs.influxdata.com/influxdb/v1.7/concepts/storage_engine).
```

Comment:
wld prob leave out refs to v1 in general...folks who've used v1 likely know, and folks who haven't prob aren't interested in learning the older system...